### ISSUE TITLE
Create consistent variable names within a data source

### ISSUE THREAD

**Issue by turnerm** [OCHA-DAP/pa-aa-toolbox#160]
It would be handy to have consistent variable names within the same data sources. Some examples to fix:

- CHIRPS monthly has `precipitation` while the daily has `prcp` (see #158)
- COD ABs are a wild west of column names

**Comment by caldwellst** [comment]
I would propose a distinction here:

- Aligning variable names between data sources is good and we should do it!
- Aligning variable names within a data source, such as for CodABs, is much more difficult. I don't think we should necessarily take this on as this also breaks if CodABs change, and I think long term there are lots of proposed changes that will more and more align these.

What do you think?

**Comment by turnerm** [comment]
Agreed that COD ABs is probably too ambitious -- they will eventually do it themselves, right??  jk. 

But I think we can tackle the CHIRPS issue. Also, to your point about between data sources, we should maybe look at how we label forecasts and lead times and all that confusing stuff between sources.
